From 790b71078d332c158a5d68ba4711ef9cffb417a1 Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 17 Sep 2014 16:24:35 -0400
Subject: [redhat] kabi: Adding symbol param_set_ulong (kernel/params.c)

Message-id: <08a4cb0b911eba5e1500bfb6ae0d3e6da3828100.1410510773.git.poros@redhat.com>
Patchwork-id: 96043
O-Subject: [PATCH RHEL7.1 155/233] kabi: Adding symbol param_set_ulong (kernel/params.c)
Bugzilla: 1071099
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

Bugzilln: 1071099
    https://bugzilla.redhat.com/show_bug.cgi?id=1071099

    Description:
    ============
    Not used directly, used by module_param internally for type ulong.

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/param_set_ulong
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/param_set_ulong
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/param_set_ulong

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/param_set_ulong b/redhat/kabi/kabi-module/kabi_ppc64/param_set_ulong
new file mode 100644
index 0000000..7c3fa0d
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/param_set_ulong
@@ -0,0 +1,2 @@
+#1-
+0x8f7014a1 param_set_ulong vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/param_set_ulong b/redhat/kabi/kabi-module/kabi_s390x/param_set_ulong
new file mode 100644
index 0000000..7c3fa0d
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/param_set_ulong
@@ -0,0 +1,2 @@
+#1-
+0x8f7014a1 param_set_ulong vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/param_set_ulong b/redhat/kabi/kabi-module/kabi_x86_64/param_set_ulong
new file mode 100644
index 0000000..7c3fa0d
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/param_set_ulong
@@ -0,0 +1,2 @@
+#1-
+0x8f7014a1 param_set_ulong vmlinux EXPORT_SYMBOL
-- 
1.7.1